gremlin-dotnet/glv/Gremlin.Net.csproj.template

```diff
@@ -25,7 +25,9 @@ limitations under the License.
   </PropertyGroup>
 
   <PropertyGroup Label="Package">
-    <Version>${projectVersion}</Version>
+    <Version>$projectVersion</Version>
+    <FileVersion>$projectVersion.0</FileVersion>
```

Comment:
`FileVersion` only accepts `major.minor.patch.build` format, we need to strip any other character like `-SNAPSHOT`.

Reply:
A good option to simplify this might be with the `VersionPrefix` and `VersionSuffix` elements:
```
<VersionPrefix>3.2.7</VersionPrefix>
<VersionSuffix>rc1</VersionSuffix>
```
Then we can use the `VersionPrefix` directly for the `FileVersion`.

Reply:
ok - fixed `<FileVersion>` - i just did some regex. i didn't quite understand the suggestion that @FlorianHockmann made, but I can adjust if you guys determine it is right to do so. at least it builds now.

Reply:
My suggestion was to replace this:
```xml
<Version>3.2.7-SNAPSHOT</Version>
<FileVersion>3.2.7.0</FileVersion>
<AssemblyVersion>3.2.0.0</AssemblyVersion>
```
by this:
```xml
<VersionPrefix>3.2.7</VersionPrefix>
<VersionSuffix>SNAPSHOT</VersionSuffix>
<FileVersion>3.2.7.0</FileVersion>
<AssemblyVersion>3.2.0.0</AssemblyVersion>
```
as the second option seems to be easier to generate, but both are completely valid and we can stay with the first option when you already managed to adapt the generation for that.

Reply:
ok - thanks. as it was pretty straightforward to do it as i did it, i'll just leave it as is with the regex solution.